pennylane/io/qualtran_io.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the adapter class for Qualtran-PennyLane interoperability.
+"""
+# pylint:disable=
+
+import pennylane as qml
+from qualtran import Bloq, CompositeBloq, Soquet, LeftDangle, Register, Side
```

Comment:
Are we adding qualtran as a dependency?

Reply:
Yes soft dependency is needed. I haven't done the import_or_skip thing yet, but fret not young lad, it will be done.